src/js/view/controls/components/settings/menu.js

```diff
@@ -19,22 +19,59 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
 
     const settingsMenuElement = createElement(SettingsMenuTemplate());
 
-    const closeOnEnter = function(evt) {
-        if (evt && evt.keyCode === 27) {
-            instance.close(evt);
-            evt.stopPropagation();
+    const nextSibiling = function(element) {
+        return element.nextElementSibling;
+    };
+
+    const previousSibling = function(element) {
+        return element.previousElementSibling;
+    };
```

Comment:
Since these functions are static and do not need access to data in `SettingsMenu`'s scope, they should be defined outside of `SettingsMenu` as named functions:

```js
function previousSibling (element) {
    return element.previousElementSibling;
}

export function SettingsMenu...
```